### PR TITLE
Add package builder images for ubuntu.

### PR DESCRIPTION
**Pull request by Ferroin**
Same as #31, just for Ubuntu this time.

### CODE REVIEW

**Comment by prologic** [comment]
Let's stick to the same naming convention:

`package-builders/Dockerfile.ubuntu1604` => `package-builders/Dockerfile.ubuntu_1604`

This is important or the `Dockerfile` in netdata/netdata#7735 doesn't work :)

**Comment by prologic** [comment]
> This is important or the `Dockerfile` in [netdata/netdata#7735](https://github.com/netdata/netdata/pull/7735) doesn't work :)

Well correct; it will; because you can `docker build -t <anything you want>` but consistency :) We can script things easier.

**Comment by prologic** [comment]
In testing Ubuntu 18.04 ...

Whilst the image itself seems [fine](https://gist.github.com/prologic/5bbbb3b657c328e2aaf16cd414d6ee2f) the build fails with:

```sh
make[1]: Entering directory '/netdata'
dh_installdocs
find . \
	-name README.md \
	-not -path './.travis/*' \
	-not -path './debian/*' \
	-not -path './contrib/*' \
	-exec cp \
	--parents \
	--target /netdata/debian/netdata/usr/share/doc/netdata/ \
	{} \;
make[1]: Leaving directory '/netdata'
   dh_installchangelogs
dh_installchangelogs: warning:     debian/changelog(l5): badly formatted heading line
LINE: -- Netdata Builder <bot@netdata.cloud>  2020-01-16
dh_installchangelogs: warning:     debian/changelog(l6): found end of file where expected more change data or trailer
dh_installchangelogs: changelog parse failure
make: *** [binary] Error 255
debian/rules:10: recipe for target 'binary' failed
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
The command '/bin/sh -c /build.sh' returned a non-zero code: 1

real	1m27.957s
user	0m0.477s
sys	0m0.507s
```

I'm not sure why yet...

**Comment by Ferroin** [comment]
> In testing Ubuntu 18.04 ...
> 
> Whilst the image itself seems [fine](https://gist.github.com/prologic/5bbbb3b657c328e2aaf16cd414d6ee2f) the build fails with:
> 
> ```shell
> make[1]: Entering directory '/netdata'
> dh_installdocs
> find . \
> 	-name README.md \
> 	-not -path './.travis/*' \
> 	-not -path './debian/*' \
> 	-not -path './contrib/*' \
> 	-exec cp \
> 	--parents \
> 	--target /netdata/debian/netdata/usr/share/doc/netdata/ \
> 	{} \;
> make[1]: Leaving directory '/netdata'
>    dh_installchangelogs
> dh_installchangelogs: warning:     debian/changelog(l5): badly formatted heading line
> LINE: -- Netdata Builder <bot@netdata.cloud>  2020-01-16
> dh_installchangelogs: warning:     debian/changelog(l6): found end of file where expected more change data or trailer
> dh_installchangelogs: changelog parse failure
> make: *** [binary] Error 255
> debian/rules:10: recipe for target 'binary' failed
> dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
> The command '/bin/sh -c /build.sh' returned a non-zero code: 1
> 
> real	1m27.957s
> user	0m0.477s
> sys	0m0.507s
> ```
> 
> I'm not sure why yet...

It's due to incorrect formatting in the changelog file. I've pushed a new commit here that partially fixes this (by using the correct date format), but for this to work we'll also need to have netdata/netdata#7763 merged

**Comment by Ferroin** [comment]
Force pushed after rebase to resolve conflict with master branch.

**Comment by Ferroin** [comment]
ANd now it fails in a different place due to this: https://bugs.launchpad.net/ubuntu/+source/debhelper/+bug/1766102

I'll update the default `VERSION` value for everything for consistency's sake.

File: package-builders/Dockerfile.ubuntu1604 [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+ENV EMAIL=$EMAIL
+ENV FULLNAME=$FULLNAME
+ENV VERSION=$VERSION
+
+# This is needed to keep package installs from prompting about configuration.
+ARG DEBIAN_FRONTEND=noninteractive
+
+ADD package-builders/debian-build.sh /build.sh
+RUN chmod +x /build.sh
```
Comment (prologic) [inline package-builders/Dockerfile.ubuntu1604]: Unnecessary if you `chmod +x` the file and commit it that way into Git :) I prefer doing it this way myself.

**Comment by Ferroin** [comment]
Holding off on merging until I've verified that 16.04 and 18.04 are working correctly.

**Pull request merged**